frontend/src/APIClients/EntityAPIClient.ts

```diff
@@ -41,7 +41,7 @@ const create = async ({
       headers: { Authorization: bearerToken },
     });
     return data;
-  } catch (error) {
```

Comment:
Would setting these error types to `unknown` rather than `any` be possible? From what I know, `unknown` is usually preferred for TS catch errors, and other BP codebases seem to use it as well.

Reply:
Updated, we are now throwing the error and using console.log to output the error, we ended up avoiding using the logger because it seems to not be ideal for the front end [(reference)](https://stackoverflow.com/questions/51110058/can-winston-logger-be-used-on-the-front-end-for-logging#:~:text=Yes%20%2D%20it%20can%20(technically)%20be%20used%20in%20the%20browser.)

Reply:
@ssreekar Makes sense!